static/favicons/safari-pinned-tab.svg

```diff
@@ -0,0 +1 @@
 3.7 13.3.5 10.1.9 16 1.9 24.5.3 2.2.7 6.2 1 9 .4 2.7.8 6.1 1.1 7.5 1.2 5.7 1.6 7.7 2.1 11 1.1 6.6 5.5 22.5 7.1 25.5.8 1.6 2.3 4.3 3.1 6 1.7 3.2 2.8 13.9 1.9 17.7-.6 2.3-.8 2.3-17.9 2.5-37.3.5-62.9.2-66.9-.6-1.3-.2-4.5-3.8-6.1-6.7-5-9.1-3.8-23.4 2.5-29.7 2-2 9.4-6 11.7-6.4 7.7-1.2 11.1-1 16.4 1.2 6.6 2.7 15.4 8.5 18.7 12.4 1.2 1.4 3.8 4.4 5.7 6.5l3.5 4-.7-4.5c-1-6.6-6.1-16.2-10.8-20.3-2.2-2-6.7-5.2-10-7.2-4.8-2.9-8.8-7.4-9.3-10.4-.1-.3-1.1-2.4-2.3-4.7-4.7-8.9-10.2-22.5-15-36.9-7.4-21.9-7.4-21.8-1.5-24.8 8.2-4.1 21.2-5.5 31.8-3.2zm161.9-.4c6.1 1.3 15.6 5.9 15.6 7.5 0 2-9.8 32.3-11 34-.4.5-1.8 4-3.1 7.7-1.3 3.8-2.7 6.8-3.1 6.8-.5 0-.8.7-.8 1.5 0 .9-.7 2.7-1.6 4-.9 1.4-1.3 2.5-.9 2.5.5 0 .1.8-.7 1.7-.8 1-2 3.2-2.7 4.9-1.5 3.8-5.3 7.9-9.2 9.9-11.4 6-19.2 16.4-20.6 28l-.5 3.9 2.8-3.4c8.2-10.1 18-17.4 26.9-20.3 4.9-1.5 7.5-1.6 14.4-.3 5.2 1 12.1 5.2 12.1 7.5 0 .6.5 1.1 1 1.1 1.8 0 3.7 5.6 4.3 12.6.7 7.8-5.3 21.2-9.6 21.5-.7 0-2.1.3-3.2.5-4.2.9-80.1.1-81.1-.9-.3-.3-.6-3.7-.6-7.4 0-5.9.5-8 3.7-14.6 2-4.2 4.1-9.4 4.6-11.5.5-2 1.2-4.4 1.4-5.2.9-2.8 3.4-14.5 4-18.5.3-2.2.8-4.7 1-5.5.4-1.6 1.5-9.3 2-15 .2-1.9.6-6.2 1-9.5.5-5.4 1.2-16.7 1.4-23.8.1-2.1 1.4-3.6 6-7 14.7-10.9 32.7-15.8 46.5-12.7zm-254.1 14.4c9.5 3.4 19.8 8.1 35.6 16.1l8.4 4.2 4.2 10.9c2.3 6 5.6 14 7.3 17.8 2.9 6.3 3.2 8.1 1.4 8.1-.4 0-2.9 1.2-5.6 2.5-10.5 5.3-19.7 18.1-21.2 29.2-.4 2.7-.9 4.9-1.3 4.9-.3.1-16 .2-34.9.3-29.4.1-34.5-.1-35.7-1.4-2.2-2.5-1.2-12.6 1.8-18.7 6.1-12 17.6-18.2 33.2-17.7 8.8.3 20.6 3.1 30.1 7.3 5.5 2.5 5.3 2.2-4.6-5.9-19.2-15.8-28-31.1-29.6-51.5-.3-3.3-.7-6.8-.9-7.8-.5-2.4 1.1-2.2 11.8 1.7zm336.2 3.4c-1.1 18.1-7.4 32.1-20.5 45.3-4.7 4.7-10.5 10.2-13 12.3-2.5 2.1-3.6 3.4-2.5 3 18-6.4 23.8-7.7 34-7.7 14.3 0 25.6 6.6 31.1 18 2.9 6.2 3.8 16.3 1.6 18.5-1 1-9.8 1.3-36.7 1.3-19.5 0-35.6-.2-35.6-.5-.1-.3-.3-1.4-.5-2.5-1.4-9.3-5.1-16.9-11-22.9-4.4-4.4-12.4-9.6-14.8-9.6-3.6-.1-3.6-1.7.3-10.6 2.1-4.9 5.1-12.5 6.6-16.9 1.5-4.4 3.1-8.5 3.7-9.2.8-1.1 31.1-16.3 32.4-16.3.3 0 2-.7 3.7-1.5 7.1-3.3 17.2-7.1 19.4-7.3 2.2-.2 2.3 0 1.8 6.6zM299 588.3c3.6 0 10.4-.3 15.2-.7l8.7-.7-.4 4.8c-.2 2.6-.6 8.9-.9 13.8-.4 5-.8 10.2-1.1 11.5-.3 1.4-.7 4.7-1 7.5-.9 7.8-4.4 24.6-6 28.7-.8 2.1-1.3 3.8-1 3.8.3 0-.6 2.2-2 5l-2.5 5h-12.6l-2.3-4.5c-3.1-6.3-7.2-21.8-8.5-32.5-.3-1.9-.5-3.5-.6-3.5-.1 0-.3-1.1-.5-2.5s-.6-4.5-.9-7c-.5-4.1-1.3-13.9-2-24.7-.2-2.3-.1-4.5.2-4.8.3-.3 3.1-.3 6.1.1 3.1.3 8.5.6 12.1.7z"/><path d="M282.4 73c-3.8 1.5-4.7 6.8-1.7 10.2 3.4 3.8 10.6.6 10.6-4.6 0-4.5-4.6-7.3-8.9-5.6zM283 100.7c-4.2 1.7-5.5 7.7-2.2 10.6 3.8 3.4 9.2 1.4 10.4-3.9 1-3.8-4.3-8.1-8.2-6.7zM320 74.8c-5.1 6.4 3.4 14 9 8 2.5-2.7 2.5-4.9.3-7.7-2.9-3.3-6.9-3.4-9.3-.3zM320.5 102.4c-3 3-3.2 5.9-.5 8.6 2.6 2.6 6.4 2.5 9-.2 2.5-2.7 2.5-4.9.3-7.6-3-3.4-5.9-3.7-8.8-.8zM294.8 285.5c-.1.1-2.5.6-5.3.9-7 .9-13.4 3.2-18.8 6.6-9.8 6.2-8.5 18 3 25.9 3.5 2.4 16.3 7.4 17 6.7.3-.3 1.7 0 3.2.5 3.5 1.4 18.9 1.3 22.6-.1 1.7-.6 3.1-1.1 3.3-1 .9.5 7.5-1.7 12.2-4 13.9-7.1 17.6-19.2 8.2-26.9-3.9-3.3-12.2-6.4-21.2-8-5.1-.8-23.4-1.4-24.2-.6z"/></svg>
```

Comment:
According to Apple's docs for the mask-icon:
> the viewBox attribute must be set to "0 0 16 16".

so it looks like this SVG isn't valid.

See https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html

Reply:
Done